superset/models/helpers.py

```diff
@@ -43,6 +43,7 @@
 import pandas as pd
 import pytz
 import sqlalchemy as sa
+import sqlglot
```

Comment:
We should not import `sqlglot` outside of `superset/sql/`, any SQL parsing should happen there.